server/dash/dashboards/ncSumComp.py

```diff
@@ -0,0 +1,407 @@
+                html.H1(id="num_days_card", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"}))
+
+        ], style=EQUAL_SPACE_STYLE.update({"width": "48.5vw"})),
+
+        # Indicator Visuals for Total number of requests and the number of days the data spans across
+        html.Div(children=[
+            html.Div([
+                html.H6("Total Number of Requests", style=CENTER_ALIGN_STYLE),
+                html.H1(id="total_req_card2", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"})),
+            html.Div([
+                html.H6("Number of Days", style=CENTER_ALIGN_STYLE),
+                html.H1(id="num_days_card2", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"}))
+
+        ], style=EQUAL_SPACE_STYLE.update({"width": "48.5vw"}))
+    ], style=EQUAL_SPACE_STYLE.update({"width": "97.5vw"})),
+
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+
+    # NC Comparison -  Request Source Bar Charts
+    html.Div(children=[
+        html.Div(dcc.Graph(id="req_source_bar_chart", style={"height": "30vh"}), style=CHART_OUTLINE_STYLE.update({
+                 "width": "48.5vw", "height": "30vh"})),
+        html.Div(dcc.Graph(id="req_source_bar_chart2", style={"height": "30vh"}), style=CHART_OUTLINE_STYLE.update({
+                 "width": "48.5vw", "margin-left": "10px", "height": "30vh"}))
+    ], style=EQUAL_SPACE_STYLE.update({"width": "97.5vw"})),
+
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+
+    # NC Comparison - Number of Requests per day Overlapping line chart
+    html.Div(dcc.Graph(id="overlay_req_time_line_chart", style={"height": "32vh", "width": "97.5vw"}), style=BORDER_STYLE.update({
+        "height": "32vh", "width": "97.5vw"}))
+])
+
+# CALLBACK FUNCTIONS
+@callback(
+    [Output("nc_dropdown_filter", "options"),
+     Output("nc_dropdown_filter", "value")],
+    Input("nc_dropdown", "value")
+)
+def generate_dynamic_filter(nc_dropdown):
+    """Enables the dashboard to show dynamic filters.
+
+    This function takes the selected neighborhood council (nc) value from the "nc_dropdown" dropdown and output a a list of available request types
+    from that neigbhorhood council.
+
+    Args:
+        nc_dropdown: A string argument automatically detected by Dash callback function when "nc_dropdown" element is selected in the layout.
+
+    Returns: 
+        nc_dropdown_filter: a list of request types available from the selected neigbhorhood council.
+        nc_dropdown_filter: placeholder value for the dropdown when nothing is selected.
+    """
+    # If no neighborhood council is selected, use all data.
+    if not nc_dropdown:
+        df = data_2020
+    else:
+        df = data_2020[data_2020["councilName"] == nc_dropdown]
+
+    req_types = sorted([n for n in set(df["typeName"])])
+    return req_types, " "
```

Comment:
Since this function always returns `" "` for the second value, I think we should just not return it.

Reply:
resolved.